app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
The idea is so the dev can query it in the console, so setting it globally on window was what came to mind first. Do you think there's a better place?